ctapipe/image/reducer.py

```diff
@@ -17,23 +22,91 @@ class DataVolumeReducer(Component):
     Base component for data volume reducers.
     """
 
-    @abstractmethod
```

Comment:
I think now we should make `DataVolumeReducer` (and thus its children like TailcutsDataVolumeReducer)  inherit from `TelescopeComponent`, and use `TelescopeParameters` for all of its config options. That allows us to set different thresholds for each telescope, which I imagine is needed?  See the implementation of `ImageCleaner` or `ImageExtractor` for examples.

Reply:
ah, I just saw @watsonjj 's comment to move that to another PR.  Ok I'll accept this then, and you should open an issue about updating it to use TelescopeComponent

Reply:
All right, thank you! I'll will make another PR then for the Telescope Component!

Reply:
No need to make another PR, just update it here

Reply:
ok then, i'll commit it here.

Reply:
I set `keep_isolated_pixels=False` for the `tailcuts_clean` like in `ImageCleaner`, since there is no BoolTelescopeParameter isn't it?

Reply:
Actually, you should just use an image cleaner component as a child component, right? That way, you would automatically expose the options of `ImageCleaner` even when those might change

Reply:
Also, adding the `BoolTelescopeParameter` should be trivial, have a look at `ctapipe/core/traits.py` , you should be able to just adapt the flaot and int versions.

Reply:
See #1281